html_css/v2/foundations/html-foundations/html-boilerplate.md

```diff
@@ -0,0 +1,147 @@
+### Introduction
+
+All HTML documents have the same basic structure or boilerplate that needs to be in place before anything useful can be done. In this lesson we will explore the different parts of this boilerplate and see how it all fits together.
+
+### Learning Outcomes
+
+- How to write the basic boilerplate for a html document
+- How to open html documents in your browser
+
+### Creating a HTML File
```

Comment:
I think I was going to go down this route initially. But I changed my mind because I thought having students build up the boilerplate each step of the way would possibly be a more engaging experience than showing them up front what the end product would look like. 

But I definitely get what you're saying about showing them the end of the road, if this lesson was longer that would be an absolute must. But because its on the short side I think it would could be fine. 

I'm definitely happy to change it if you have strong opinion on this one mate, but wanted to share my perspective first.